app/controllers/users_controller.rb

```diff
@@ -996,11 +996,15 @@ def destroy
   end
 
   def ignore
-    raise Discourse::NotFound unless SiteSetting.ignore_user_enabled
+    unless SiteSetting.ignore_user_enabled
```

Comment:
I actually preferred this on one line. Also it reads better if you do `ignore_user_enabled?`